src/components/input/Select.tsx

```diff
@@ -99,11 +99,10 @@ function SelectOption<T>({
     <li
       className={classnames(
         'w-full ring-inset outline-none rounded-none select-none',
-        'border-t first:border-t-0 whitespace-nowrap',
+        'px-1 mb-1 first:mt-1 whitespace-nowrap group',
```

Comment:
I have decided to implement spacing using margins and padding on the options, so that it's easier to eventually include the search boxes.

Reply:
This is fine, but can you clarify how this makes adding the search boxes easier?

Reply:
Yes.

I initially considered adding a small padding to the surrounding listbox, but according to the designs, the search box container should span to the very sides of said listbox.

![image](https://github.com/user-attachments/assets/d75f2d0e-e69a-4684-8d1b-43b8d4a51fe3)

Adding the padding to every individual option instead, ensures that will be eventually possible.